Sofa/Component/IO/Mesh/src/sofa/component/io/mesh/MeshSTLLoader.cpp

```diff
@@ -43,10 +43,13 @@ static int MeshSTLLoaderClass = core::RegisterObject("Loader for the STL file fo
 
 //Base VTK Loader
 MeshSTLLoader::MeshSTLLoader() : MeshLoader()
-    , _headerSize(initData(&_headerSize, 80u, "headerSize","Size of the header binary file (just before the number of facet)."))
-    , _forceBinary(initData(&_forceBinary, false, "forceBinary","Force reading in binary mode. Even in first keyword of the file is solid."))
+    , d_headerSize(initData(&d_headerSize, 80u, "d_headerSize", "Size of the header binary file (just before the number of facet)."))
```

Comment:
```suggestion
    , d_headerSize(initData(&d_headerSize, 80u, "headerSize", "Size of the header binary file (just before the number of facet)."))
```
Be careful in all of your PRs that you don't change the name of the Data.